stable/postgresql/templates/svc.yaml

```diff
@@ -16,6 +16,10 @@ spec:
   {{- if and .Values.service.loadBalancerIP (eq .Values.service.type "LoadBalancer") }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
+  {{- if .Values.service.loadBalancerSourceRanges }}
```

Comment:
Shouldn't you check if the service.type is a **LoadBalancer** too?

```yaml
{{- and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerSourceRanges }}
```

Reply:
Yes, better 👍 I changed to check.